tests/gitlab/get_mfem_uberenv.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+uberenv_url="https://github.com/mfem/mfem-uberenv.git"
+uberenv_ref="c9f95ab4d51c283a5e18d8f850a52935a402f679"
+
+
+[[ ! -d scripts/uberenv ]] && git clone ${uberenv_url} tests/uberenv
```

Comment:
Is this expected to be run from the top-level MFEM repo? If so, mention that in the beginning of the file (or add a help option as in `../scripts`).

Reply:
Done (not using help function... yet?)